README.md

```diff
@@ -281,9 +281,11 @@ end
 
 List of configurable settings with their default values.
 
-##### Redis
+##### Token store
 
-Default token store configurations
+In order to configure token store you should either set up a concrete instance of store adapter (`JWTSessions.token_store = JWTSessions::StoreAdapters::RedisStoreAdapter.build(redis_url: 'redis://127.0.0.1:6379/0')`) or pass a symbol and optional options (`JWTSessions.token_store = :redis, { redis_url: 'redis://127.0.0.1:6379/0', token_prefix: 'prefix' }`). Currently supported stores are `:redis` (`RedisStoreAdapter`) and `:memory` (`MemoryStoreAdapter`). Please note, that if you want to use Redis as a store then you should have `redis` gem listed in your Gemfile. If you won't configure the adapter explicitly, this gem will try to load `redis` and use it, otherwise it would fallback to a `memory` adapter.
+
```

Comment:
Besides `redis_url` there're also such settings as `redis_host`, `redis_port`, `redis_db_name`. 
Taking all that into account it feels like the configuration becomes super confusing.

Is it just me, or a basic setting notation like this
```ruby
JWTSessions.token_store = JWTSessions::StoreAdapters::RedisStoreAdapter.build(redis_url: 'redis://127.0.0.1:6379/0')`
```
looks overwhelming?
Let's keep only `JWTSessions.token_store = :redis, options`, what do you think?

Also it's super minor, but... "optional options".

Reply:
> Besides redis_url there're also such settings as redis_host, redis_port, redis_db_name. Taking all that into account it feels like the configuration becomes super confusing.

Yeah, let's make the adapter to accept these options too, but keep backward compatibility as usual

> Let's keep only JWTSessions.token_store = :redis, options, what do you think?
Yeah, I like that, not sure if anyone would prefer writing the exact classes

and yes, optional options🤦‍♂️

Reply:
addressed all the mentioned issues